lib/view_component/form/builder.rb

```diff
@@ -52,7 +52,7 @@ def submit(value = nil, options = {})
           value = nil
         end
         value ||= submit_default_value
-        render_component(:submit, self, @object_name, value, options)
```

Comment:
Not sure about removing `@object_name` here

Reply:
Sounds okay